Makefile

```diff
@@ -0,0 +1,28 @@
+########################################################################################
+
+# This Makefile generated by GoMakeGen 0.5.0 using next command:
+# gomakegen --output Makefile .
```

Comment:
`Makefile` is default output file in gomakegen. You can execute gomakegen without any options.